### PR TITLE
[DDW-685] Stake Pools - Adjustments

### PR DESCRIPTION
**Pull request by daniloprates**
This PR adds fine adjustments to the Stake Pools screen.

## Todos

* [X] https://docs.google.com/document/d/10UwMUoDBalrRS4QvLZq2dtXJ42ICdpfgDPIUo3Fcxeg/edit#

* **Tooltip positioning** and **Search/Filtering** will be handled on different PR's
* **[Temporary Storybook URL](https://daniloprates.github.io/daedalus/?selectedKind=Staking&selectedStory=Stake%20Pools&full=0&addons=1&stories=1&panelRight=0&addonPanel=storybooks%2Fstorybook-addon-knobs)**, while Netifly is failing.

## Screenshots

image

---

## Testing Checklist

- [Slack QA thread](https://input-output-rnd.slack.com/archives/GGKFXSKC6/p1560368504017400)
- [ ] Test

---

### Basics

- [ ] PR has been assigned and has appropriate labels (`feature`/`bug`/`chore`, `release-x.x.x`)
- [ ] PR is updated to the most recent version of the target branch (and there are no conflicts)
- [ ] PR has a good description that summarizes all changes and shows some screenshots or animated GIFs of important UI changes
- [ ] CHANGELOG entry has been added to the top of the appropriate section (*Features*, *Fixes*, *Chores*) and is linked to the correct PR on GitHub
- [ ] Automated tests: All acceptance and unit tests are passing (`yarn test`)
- [ ] Manual tests (minimum tests should cover newly added feature/fix): App works correctly in *development* build (`yarn dev`)
- [ ] Manual tests (minimum tests should cover newly added feature/fix): App works correctly in *production* build (`yarn package` / CI builds)
- [ ] There are no *flow* errors or warnings (`yarn flow:test`)
- [ ] There are no *lint* errors or warnings (`yarn lint`)
- [ ] There are no *prettier* errors or warnings (`yarn prettier:check`)
- [ ] There are no missing translations (running `yarn manage:translations` produces no changes)
- [ ] Text changes are proofread and approved (Jane Wild / Amy Reeve)
- [ ] Japanese text changes are proofread and approved (Junko Oda)
- [ ] UI changes look good in all themes (Alexander Rukin)
- [ ] Storybook works and no stories are broken (`yarn storybook`)
- [ ] In case of dependency changes `yarn.lock` file is updated

### Code Quality
- [ ] Important parts of the code are properly commented and documented
- [ ] Code is properly typed with flow
- [ ] React components are split-up enough to avoid unnecessary re-renderings
- [ ] Any code that only works in main process is neatly separated from components

### Testing
- [ ] New feature/change is covered by acceptance tests
- [ ] New feature/change is manually tested and approved by QA team
- [ ] All existing acceptance tests are still up-to-date
- [ ] New feature/change is covered by Daedalus Testing scenario
- [ ] All existing Daedalus Testing scenarios are still up-to-date

### After Review
- [ ] Merge the PR
- [ ] Delete the source branch
- [ ] Move the ticket to `done` column on the YouTrack board
- [ ] Update Slack QA thread by marking it with a green checkmark

### CODE REVIEW

**Comment by DominikGuzei** [comment]
@daniloprates Netlify build is fixed 👍

**Comment by daniloprates** [comment]
> @daniloprates Netlify build is fixed 👍

@DominikGuzei 💛

**Comment by a-rukin** [comment]
image
left distance != right distance (due to the scroll) is this an issue for this PR ?

**Comment by a-rukin** [comment]
image
this bug relates to position, right?

**Comment by a-rukin** [comment]
image
close cross is bigger than 8px
should we decrease size?

**Comment by a-rukin** [comment]
image
for controlled stake should be different "color-logic", but this is not issue of this PR, just mentioning

**Comment by a-rukin** [comment]
Rest is very good! 👍

**Comment by daniloprates** [comment]
> for controlled stake should be different "color-logic", but this is not issue of this PR, just mentioning

@a-rukin this issue is for this PR. What should be the logic in this case?

**Comment by a-rukin** [comment]
> @a-rukin this issue is for this PR. What should be the logic in this case?
I'm not completely sure, but we are aiming for all the stake be separated equally between pools 
so I would assume that the perfect (green) percentage of stake is (1 / number_of_stakes)% 
something like 90% of stake should be red -> monopoly

**Comment by daniloprates** [comment]
> > @a-rukin this issue is for this PR. What should be the logic in this case?
> 
> I'm not completely sure, but we are aiming for all the stake be separated equally between pools
> so I would assume that the perfect (green) percentage of stake is (1 / number_of_stakes)%
> something like 90% of stake should be red -> monopoly

Cool, I'll create a separated method for this algorithm and we can improve it once we get new information.

**Comment by daniloprates** [comment]
> this bug relates to position, right?

I'll remove the fixed Search component until we find a better solution. ATM it's taking too much space.

**Comment by daniloprates** [comment]
> left distance != right distance (due to the scroll) is this an issue for this PR ?

@a-rukin in this case, I can't just ignore the right margin, as it would make it even when there's the scrollbar, but uneven when there is not.

Let's leave this one like this and I'll open a card for us to study handling the scrollbar in a better way. Maybe we can apply one of those Javascript solutions globally in the app.

**Comment by a-rukin** [comment]
> Let's leave this one like this and I'll open a card for us to study handling the scrollbar in a better way. Maybe we can apply one of those Javascript solutions globally in the app.

Sure 👌

**Comment by daniloprates** [comment]
Hey reviewers, all the requests were made and are available here: https://daniloprates.github.io/daedalus/?selectedKind=Staking&selectedStory=Stake%20Pools&full=0&addons=1&stories=1&panelRight=0&addonPanel=storybooks%2Fstorybook-addon-knobs

File: source/renderer/app/components/staking/stake-pools/StakePoolTooltip.scss [hunk 1]
```diff
@@ -187,18 +207,30 @@
 .table {
   @extend %regularText;
   margin-bottom: 14px;
+  display: flex;
```
Comment (nikolaglumac) [inline source/renderer/app/components/staking/stake-pools/StakePoolTooltip.scss]: @daniloprates please sort these CSS rules alphabetically 🙏

**Comment by nikolaglumac** [comment]
@MarcusHurney please review this one too 🙏

File: source/renderer/app/components/staking/stake-pools/StakePoolTooltip.scss (hunk 1 above)
Comment (nikolaglumac) [reply source/renderer/app/components/staking/stake-pools/StakePoolTooltip.scss]: @daniloprates fixed in: https://github.com/input-output-hk/daedalus/pull/1420/commits/425f483ad1bfb715ab18770650d090921d7ef431

**Comment by nikolaglumac** [comment]
@daniloprates there are some issues with AppVeyor. @cleverca22 will help us resolve them in the scope of this PR.

**Pull request merged**